consensus/misc/eip1559/eip1559.go

```diff
@@ -47,48 +49,53 @@ func VerifyEip1559Header(config *params.ChainConfig, parent, header *types.Heade
 
 // CalcBaseFee calculates the basefee of the header.
 func CalcBaseFee(config *params.ChainConfig, parent *types.Header) *big.Int {
-	return big.NewInt(0)
+	blockNumber := new(big.Int).Add(parent.Number, common.Big1)
 
-	/*
-		We temporarily set the base fee to 0, the following logic may be enabled in
-		the future hardfork.
+	if !config.IsShanghai(blockNumber) {
```

Comment:
Let's use a Ronin hardfork name instead of Shanghai. This feature isn't actually part of the Shanghai hardfork.